astropy/modeling/fitting.py

```diff
@@ -1345,7 +1345,7 @@ def _fitter_to_model_params(model, fps):
             if model.tied[name]:
                 value = model.tied[name](model)
                 slice_ = param_metrics[name]['slice']
-                parameters[slice_] = value
+                model.parameters[slice_] = value
```

Comment:
Just for clarification. I'm puzzling over why these two versions result in different behavior. Normally a slice is a view (model.parameters returns a slice), so assignment to a slice of a slice should be a view (for a 1d array with contiguous slices). The only case that seems to result in different behavior is if _slice is relative to model._parameters array, not the slice of it which is returned. Is that what is breaking here?